distribution/sdk/sample-cometd/java-cli-example/src/main/java/org/codice/ddf/cli/modules/GlobalOptions.java

```diff
@@ -33,9 +33,6 @@
     @Option(name = {"-n", "--no-validation"}, title = "Disable Validation", description = "Disables validation of SSL certificates")
     private boolean disableValidation = true;
 
-    @Option(name = "--help", title = "Help", description = "Provides help for the current command")
-    private boolean helpOption = false;
```

Comment:
@shaundmorris I'm actually not sure it needs to be there. I think the help option isn't implemented at the moment so you can remove it